atomic

```diff
@@ -160,7 +160,7 @@ def create_parser(help_text):
     helpp.set_defaults(_class=AtomicHelp, func='help')
     helpp.add_argument("image", help=_("Image ID or name"))
 
-    if os.path.exists("/usr/bin/rpm-ostree"):
+    if os.path.exists("/usr/bin/rpm-ostree") and os.path.exists("/ostree"):
```

Comment:
I'd look for `/run/ostree-booted` actually.  See https://github.com/projectatomic/commissaire/issues/194#issuecomment-241588340